polaris/polaris/integrations/forms.py

```diff
@@ -26,6 +26,10 @@ class TransactionForm(forms.Form):
 
     def clean_amount(self):
         """Validate the provided amount of an asset."""
+        # TODO: do we want all amounts to be rounded?
```

Comment:
Lets keep PRs focused on one change.

But i don't think we should round 1.123USD to 1.12USD, you should be able to trade fractions of a cent.

Reply:
Do banks allow that? What if someone enters 1.123 but can only transfer 1.12 through their bank?